### PR TITLE
(Draft) Add languages to account creation

### LINKED ISSUE #398: Add languages to account creation
### Summary and Background
Account creation languages field for admin and volunteers

### User Story
As an Admin/Volunteer, I want to enter my languages into the platform, so the Super Admin can contact me for appropriate roles
As a Super Admin, I want to see volunteer/admin's languages, so I can contact/refer to volunteers/admins for appropriate roles.

### Solution (Core Elements)
Create language field to account creation personal info page, accessed from account creation landing page (#340).

This field is to be added to both volunteer and admin account creation.

### Mockups
Volunteer (left) and admin (right)

Other designs: 
https://www.figma.com/file/tOK6AjkaRZ1NBXo191D64C/Sistering-%2F-User-Authentication-Process?node-id=390%3A1975

Technical notes:
- Extend changes in https://github.com/uwblueprint/sistering/pull/381 to include language dropdown
- You can choose to use a hard-coded sample skill constant for the dropdown, this will be replaced with a query call in a future ticket

### PR DESCRIPTION
**Pull request by AdrianaCeric**
## Ticket link

Closes #398 

## Implementation description
* Add UI
* Added some states

## Steps to test
1. Go to [http://localhost:3000/new-account](http://localhost:3000/new-account)

## What should reviewers focus on?
* Please give feedback on what to do next! Or if you notice something I did wrong. I am mostly emulating work done for skill selection and dropdown in `http://localhost:3000/new-account`

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/user/AccountForm.tsx [hunk 1]
```diff
@@ -48,6 +51,15 @@ const SKILLS = gql`
...
 
+const LANGUAGES = gql`
+  query BranchManagerModal_Languages {
+    languages {
+      id
+      name
+    }
+  }
+`;
+
```
Comment (AdrianaCeric) [inline frontend/src/components/user/AccountForm.tsx:62]: I don't actually know if this query works? I forgot to not include it in this draft

Comment (MatoPlus) [reply frontend/src/components/user/AccountForm.tsx:62]: YEa I don't think this works, just use the static array constant in the frontend types

Comment (MatoPlus) [inline frontend/src/components/user/AccountForm.tsx:62]:
```suggestion
```

File: frontend/src/components/user/AccountForm.tsx [hunk 2]
```diff
@@ -81,6 +95,13 @@ const AccountForm = ({
     },
   });
 
+  useQuery<LanguageQueryResponse>(LANGUAGES, {
+    fetchPolicy: "cache-and-network",
+    onCompleted: (data) => {
+      setSkills(data.languages);
+    },
+  });
```
Comment (MatoPlus) [inline frontend/src/components/user/AccountForm.tsx:103]: I think this is the wrong query to set skills? You're trying to set skills as languages here

Comment (AdrianaCeric) [reply frontend/src/components/user/AccountForm.tsx:103]: Thanks!

**Pull request closed without merging**